client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -24,12 +25,27 @@ import Gridicon from 'calypso/components/gridicon';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import { resendVerificationEmail } from 'calypso/state/email-forwarding/actions';
 
-const MailboxListHeader = ( { children } ) => {
+const getListHeaderTextForAccountType = ( accountType, translate ) => {
+	if ( accountType === EMAIL_ACCOUNT_TYPE_FORWARD ) {
+		return translate( 'Forwarded mailboxes', {
```

Comment:
Are we sure we want to introduce yet another term? In most places we refer to those as `email forwards` or just `forwards`.

Reply:
I was pulling this one directly from the designs -- @saygunnyc, could you weigh in on which term(s) we should use here?

Reply:
First of all, I want to apologize in advance for any inconsistencies in the terms that you might have seen in the designs. And thank you for the ping!

- I am for creating consistent terminology everywhere!
- This was what I've been intending to go for, but I'm **REALLY** open to better suggestions.
Professional Email – Mailbox / Mailbox(es) / Add Another Mailbox
Google Workspace – User / User(s) / Add Another User
Email Forwarding – Forwarding / Forwarding(s) / Add Another Forwarding
- I haven't had the chance to reply to your [comment](https://wp.me/pcDL0W-cl#comment-513) here @daledupreez and it seems like we haven't heard from the emailguild or anyone else too. 
- I neither want to dictate us into a decision nor want to turn this into a bigger issue but perhaps we should try to do a quick survey around the terminology? Or what do u guys think would be the best way to resolve this? 

Lemme know your thoughts!!

Reply:
Maybe us 3 could try to do a sync chat, or a huddle to try to align as a smaller group first? I'm AFK tomorrow but open to any other time suggestions if we're to do this.

Reply:
For now, I am going to use "Email forwards" (as it's a term we have in some longer sentences), and we can iterate or improve early next week.